packages/arb-token-bridge-ui/src/components/TransferPanel/DepositConfirmationDialog.tsx

```diff
@@ -0,0 +1,250 @@
+import { Fragment, useState } from 'react'
+import { useCopyToClipboard } from 'react-use'
+import { Tab, Dialog as HeadlessUIDialog } from '@headlessui/react'
+import { XIcon, ExternalLinkIcon } from '@heroicons/react/outline'
+
+import { Dialog, UseDialogProps } from '../common/Dialog'
+import { Transition } from '../common/Transition'
+import { ExternalLink } from '../common/ExternalLink'
+import {
+  CanonicalTokenName,
+  CanonicalTokensBridgeInfo,
+  getFastBridges,
+  FastBridgeName
+} from '../../util/fastBridges'
+import { TabButton } from '../common/Tab'
+import { useAppState } from '../../state'
+import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
+import { getNetworkName, isNetwork } from '../../util/networks'
+import { ReactComponent as CustomClipboardCopyIcon } from '../../assets/copy.svg'
+import { trackEvent } from '../../util/AnalyticsUtils'
+
+function FastBridgesTable() {
```

Comment:
Is this table the same as the one in the withdrawal confirmation dialog? If that's the case, we could extract it to a common component.

Reply:
yeah good point, I'll do that